src/triage/component/results_schema/schema.py

```diff
@@ -140,22 +141,38 @@ class ModelGroup(Base):
 class ListPrediction(Base):
 
     __tablename__ = "list_predictions"
```

Comment:
I wonder if it might be a little more natural to make this `production.predictions` to keep consistent with the `train_results` and `test_results` tables?

Reply:
I agree